### PR TITLE
Improve Neo4j installation instructions

### PR DESCRIPTION
**Pull request by dongbohu**
@JDRomano2 I tried to install and configure Neo4j yesteray on my local computer recently and found that the instructions in `README.md` are hard to follow, and some of them seem obsolete. 

This PR is based on my experience when installing Neo4J Desktop 1.4.5. Some of the long lines were split into multiple lines by my editor.

### CODE REVIEW

**Comment by dongbohu** [comment]
I have no idea why Travis build failed.

**Review by JDRomano2 — APPROVED** [review]
These are all good README edits. I will also continue to improve documentation as ComptoxAI takes better shape. Please note that we will be moving to Github Actions and discontinuing TravisCI, since the tests will require Neo4j to build and Travis does not seem to play well with Neo4j.
